src/Tvheadend.cpp

```diff
@@ -367,6 +367,39 @@ PVR_ERROR CTvheadend::GetChannels(bool radio, kodi::addon::PVRChannelsResultSet&
   return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR CTvheadend::GetChannelStreamProperties(const kodi::addon::PVRChannel& channel, std::vector<kodi::addon::PVRStreamProperty>& properties)
+{
+  CLockObject lock(m_mutex);
+
+  if (!Settings::GetInstance().GetStreamingHTTP())
+    return PVR_ERROR_NO_ERROR;
+
+  auto it = m_channels.find(channel.GetUniqueId());
+  if (it == m_channels.end())
```

Comment:
As  `it` is only used once, there's no need for a variable. 
=> `if (m_channels.find(channel.GetUniqueId() == m_channels.end())`

Reply:
`it` is used again in line 391.

Reply:
Ah, right.